### PR TITLE
Switch to using a single version of the Prismic client in the content app

### PR DESCRIPTION
**Pull request by alexwlchan**
This is following the instructions from Prismic's migration guide:
https://prismic.io/docs/technologies/prismic-client-v6-migration-guide

We still have a few other versions of old Prismic libraries hanging around (e.g. prismic-reactjs, prismic-dom). Upgrading those is sufficiently complicated as to warrant its own patch.

For #7363 and #7595.

### CODE REVIEW

File: content/webapp/app.ts [hunk 1]
```diff
@@ -20,6 +19,7 @@ import {
 } from '@weco/common/services/prismic/hardcoded-id';
 import { Periods } from '@weco/common/model/periods';
 import linkResolver from './services/prismic/link-resolver';
+import * as prismic from 'prismic-client-beta';
```
Comment (davidpmccormick) [inline content/webapp/app.ts:22]: is the plan to move everything to `prismic-client-beta`, then switch all of those out for v6 of `@prismicio/client` (which presumably has the same API)?

File: content/webapp/app.ts [hunk 2]
```diff
@@ -124,15 +124,11 @@ const appPromise = nextApp
 
     router.get('/preview', async ctx => {
       // Kill any cookie we had set, as it think it is causing issues.
-      ctx.cookies.set(Prismic.previewCookie);
+      ctx.cookies.set('io.prismic.preview');
```
Comment (davidpmccormick) [inline content/webapp/app.ts:127]: should this be [`prismic.cookie.preview`](https://prismic.io/docs/technologies/prismic-client-v6-migration-guide#replace-previewcookie-with-cookie.preview)?

**Review by davidpmccormick — APPROVED** [review]

File: content/webapp/app.ts (hunk 1 above)
Comment (alexwlchan) [reply content/webapp/app.ts:22]: Yup!

File: content/webapp/app.ts (hunk 2 above)
Comment (alexwlchan) [reply content/webapp/app.ts:127]: Hmm. I got this value by logging the value of `Prismic.previewCookie` in a local build. Let's unset both of them, just to be safe.

**Pull request merged**